docs/src/devdocs/interpolations.md

```diff
@@ -22,30 +61,136 @@ Ferrite.reference_shape_hessians_gradients_and_values!
 Ferrite.shape_value_type(ip::Interpolation, ::Type{T}) where T<:Number
 ```
 
-### Required methods to implement for all subtypes of `Interpolation` to define a new finite element
+## [How to implement a new interpolation, `QTI`](@id devdocs-howto_new-interpolation)
+As an example, we will implement a `Q`uadratic `T`riangle `I`nterpolation
+(`QTI`), corresponding to `Lagrange{RefTriangle, 2}()`. This is described
+in the devdocs as the API is not considered fully stable yet.
 
-Depending on the dimension of the reference element the following functions have to be implemented
+```@example InterpolationExample
+using Ferrite # hide
+using Test #hide
+struct QTI <: ScalarInterpolation{RefTriangle, 2}
+end
+ip_qti = QTI() # hide
+ip_lag = Lagrange{RefTriangle, 2}() # hide
+nothing # hide
+```
+Before we jump in and define the shape functions associated with this
+interpolation, we first need to consider the `RefTriangle` entities,
 
-```@docs
-Ferrite.reference_shape_value(::Interpolation, ::Vec, ::Int)
-Ferrite.vertexdof_indices(::Interpolation)
-Ferrite.dirichlet_vertexdof_indices(::Interpolation)
-Ferrite.facedof_indices(::Interpolation)
-Ferrite.dirichlet_facedof_indices(::Interpolation)
-Ferrite.facedof_interior_indices(::Interpolation)
-Ferrite.edgedof_indices(::Interpolation)
-Ferrite.dirichlet_edgedof_indices(::Interpolation)
-Ferrite.edgedof_interior_indices(::Interpolation)
-Ferrite.volumedof_interior_indices(::Interpolation)
-Ferrite.getnbasefunctions(::Interpolation)
-Ferrite.reference_coordinates(::Interpolation)
-Ferrite.is_discontinuous(::Interpolation)
-Ferrite.adjust_dofs_during_distribution(::Interpolation)
-Ferrite.mapping_type
+```@docs; canonical=false
+RefTriangle
 ```
 
-for all entities which exist on that reference element. The dof functions default to having no
-dofs defined on a specific entity. Hence, not overloading of the dof functions will result in an
-element with zero dofs. Also, it should always be double checked that everything is consistent as
-specified in the docstring of the corresponding function, as inconsistent implementations can
-lead to bugs which are really difficult to track down.
+For this particular interpolation, we have one degree of freedom associated
+with each vertex, and one degree of freedom associated with each edge.
+Following the Ferrite numbering rules, we start by enumerating the
```

Comment:
See below if it is actually missing or if my day was just too long to find it :)
```suggestion
Following the [Ferrite numbering rules](<add ref to page here>), we start by enumerating the
```

Reply:
https://ferrite-fem.github.io/Ferrite.jl/stable/topics/degrees_of_freedom/#Ordering-of-Dofs :stuck_out_tongue_winking_eye: